services/noop/service.go

```diff
@@ -30,46 +32,81 @@ import (
 const logPrefix = "[service-noop] "
 
 // NewManager creates new instance of Noop service
-func NewManager() *Manager {
-	return &Manager{}
+func NewManager(locationResolver location.Resolver, ipResolver ip.Resolver) *Manager {
+	return &Manager{
+		locationResolver: locationResolver,
+		ipResolver:       ipResolver,
+	}
 }
 
 // Manager represents entrypoint for Noop service
 type Manager struct {
-	fakeProcess sync.WaitGroup
+	fakeProcess      sync.WaitGroup
+	locationResolver location.Resolver
+	ipResolver       ip.Resolver
+	isStarted        bool
 }
 
 // Start starts service - does not block
 func (manager *Manager) Start(providerID identity.Identity) (dto_discovery.ServiceProposal, session.ConfigProvider, error) {
+	sessionConfigProvider := func() (session.ServiceConfiguration, error) {
+		return nil, nil
+	}
+
+	if manager.isStarted {
+		return dto_discovery.ServiceProposal{}, sessionConfigProvider, nil
+	}
+
 	manager.fakeProcess.Add(1)
```

Comment:
We prob should not use 'fake' wording anymore here.

Reply:
Agreed